website/docs/tutorials/06-deploying/02-migrations/declare.md

```diff
@@ -1,13 +1,18 @@
 # `declare`
 
 ```python
-def declare(contract_path: str) -> DeclaredContract:
+def declare(
+    contract_path: str,
+    *,
+    config: Optional[CheatcodeNetworkConfig] = None
```

Comment:
It would be nice to explain that config must be pass as a keyword argument. The example below could demonstrate that.